src/validacoes/schemaCliente.js

```diff
@@ -0,0 +1,54 @@
+const joi = require('joi');
+
+const schemaCliente = joi.object({
+    nome: joi.string().required().messages({
+        'any.required': 'O campo nome é obrigatório.',
+        'string.empty': 'O campo nome é obrigatório.',
+        'string.base': 'O campo nome deve possuir um formato válido.'
+    }),
+
+    email: joi.string().required().email().messages({
+        'any.required': 'O campo email é obrigatório.',
+        'string.empty': 'O campo email é obrigatório.',
+        'string.base': 'O campo email deve possuir um formato válido.',
+        'string.email': 'O campo email deve possuir um formato válido.'
+    }),
+
+    cpf: joi.string().required().min(11).max(11).messages({
+        'any.required': 'O campo cpf é obrigatório.',
+        'string.empty': 'O campo cpf é obrigatório.',
+        'string.base': 'O campo cpf deve possuir um formato válido.',
+        'string.min': 'O campo cpf deve ter no mínimo 11 caracteres',
+        'string.max': 'O campo cpf não aceita caracteres especiais.'
```

Comment:
Ao invés de dizer que o campo cpf deve ter no mínimo 11 caracteres, acho que ficaria mais claro para o usuário se a mensagem dissesse que o campo deve ter exatamente 11 caracteres numéricos.

```suggestion
        'string.min': 'O campo cpf deve ter 11 caracteres numéricos.',
        'string.max': 'O campo cpf deve ter apenas os 11 caracteres numéricos.'
```